static/app/views/settings/projectIssueGrouping/index.tsx

```diff
@@ -58,15 +60,18 @@ class ProjectIssueGrouping extends AsyncView<Props, State> {
     const {groupingConfigs} = this.state;
     const {organization, project, params, location} = this.props;
     const endpoint = `/projects/${organization.slug}/${project.slug}/`;
-    const access = new Set(organization.access);
+
+    const access = new Set(organization.access.concat(project.access));
```

Comment:
this seems a bit weird but i'll roll with it. i guess you have to pass all the scopes into the json form